frontend/src/pages/groupSettings/GroupSettings.tsx

```diff
@@ -77,70 +64,34 @@ const GroupSettings: React.FC = () => {
               e.preventDefault();
             }}
           >
-            <FormGroup fieldId="admin-groups" label={`Data Science administrator groups`}>
-              <Text>{`Select the OpenShift groups that contain all Data Science administrators.`}</Text>
-              <MenuOptionMultiSelect
-                initialState={groupSettings.adminGroups}
-                onChange={(newState) => handleMenuItemSelection(newState, GroupsConfigField.ADMIN)}
-              />
-              {!groupSettings.errorAdmin && (
-                <>
-                  <HelperText>
-                    <HelperTextItem variant="indeterminate">{CARD_FOOTER}</HelperTextItem>
-                  </HelperText>
-                  <Hint>
-                    <HintBody>
-                      All cluster admins are automatically assigned as Data Science administrators.
-                    </HintBody>
-                  </Hint>
-                </>
-              )}
-              {groupSettings.errorAdmin && (
-                <Alert
-                  isInline
-                  variant="warning"
-                  title={`Group no longer exists`}
-                  actionClose={
-                    <AlertActionCloseButton
-                      onClose={() => {
-                        setGroupSettings({ ...groupSettings, errorAdmin: undefined });
-                      }}
-                    />
-                  }
-                >
-                  <p>{groupSettings.errorAdmin}</p>
-                </Alert>
-              )}
-            </FormGroup>
+            <FormGroupSettings
+              title="Data Science administrator groups"
+              body="Select the OpenShift groups that contain all Data Science administrators."
+              footer="View, edit, or create groups in OpenShift under User Management"
+              fieldId="admin-groups"
+              groupsField={GroupsConfigField.ADMIN}
+              items={groupSettings.adminGroups}
+              error={groupSettings.errorAdmin}
+              hintText="All cluster admins are automatically assigned as Data Science administrators."
```

Comment:
I wonder if we couldn't just put this in your component under a condition of `groupsField === GroupConfigField.ADMIN` -- just to reduce the vast property passing.